### PR TITLE
Home: Show latest news and release post

### PR DESCRIPTION
**Pull request by Kissaki**
Both have validity and are important, a post may be published in both categories in close time.
We want to promote both kinds.

Latest release remains valid and interesting to users landing on home.
Latest news is also a good entry and nudge that we have them even if older.

image
image

### CODE REVIEW

**Review by Krzmbrzl — APPROVED** [review]

File: hugo/layouts/index.html [hunk 1]
```diff
@@ -3,20 +3,34 @@
...
-		{{ range $page }}
-			<div class="markdown-body">
-				<h3><a href="{{ .RelPermalink }}">{{ .Title }}</a></h3>
-				{{ partial "blog/teaser" . }}
+	{{ $blogpages := (.Site.GetPage "blog").Pages }}
+	{{ $news := where $blogpages ".Params.categories" "intersect" (slice "News") }}
+	{{ $releases := where $blogpages ".Params.categories" "intersect" (slice "Release") }}
+	<div id="home-newsblock">
+		<div>
+			<h2>Latest News</h2>
```
Comment (davidebeatrici) [inline hugo/layouts/index.html:11]:
```suggestion
			<h2>Latest news</h2>
```

File: hugo/layouts/index.html [hunk 2]
```diff
@@ -3,20 +3,34 @@
...
+						{{ partial "blog/teaser" . }}
+					</div>
+				{{ end }}
 			</div>
-		{{ else }}
-			<p>No news has been published yet.</p>
-		{{ end }}
+		</div>
+		<div>
+			<h2>Latest Release</h2>
```
Comment (davidebeatrici) [inline hugo/layouts/index.html:23]:
```suggestion
			<h2>Latest release</h2>
```

**Review by davidebeatrici — APPROVED** [review]

File: hugo/layouts/index.html (hunk 1 above)
Comment (Krzmbrzl) [reply hugo/layouts/index.html:11]: I think as a header I prefer the stylized capitalization of all words.

Comment (Kissaki) [reply hugo/layouts/index.html:11]: Yeah, me too

**Comment by Kissaki** [comment]
Thanks for the reviews

**Pull request merged**